src/Lifi.ts

```diff
@@ -213,15 +265,29 @@ class LIFI {
     }
   }
 
+  /**
+   * Get a list of active routes.
```

Comment:
It is not just any list of active routes, it is THE list of active routes
```suggestion
   * Get the active routes.
```